README.md

```diff
@@ -17,6 +17,15 @@ You need [Node.js](https://nodejs.org/) (> 8.1) to run this bot.
 4. Copy `example.config.js` to `config.js` and edit it.
 5. Start the bot via `npm start`.
 
+## Setup with Docker
+You need to have [docker](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-from-a-package) installed on your machine. 
+
+1. Create a bot via [@BotFather](https://t.me/BotFather) and grab a **token**.
+2. Clone this repository or [download zip](https://github.com/TheDevs-Network/the-guard-bot/archive/master.zip).
+3. Copy `example.config.js` to `config.js` and edit it.
+4. Run `docker build -t guards_bot .` to build image.
```

Comment:
Minor thing but why this is `guards_bot` and not `guard_bot` or `the_guard_bot` or something that matches the bot's name?

Reply:
@poeti8 It can be anything you want. `-t` option in used to create a tag so you can refer the image with that tag instead of it's hash. You can change it to `the_guard_bot` and it'll work just fine.

Reply:
Ahh I see, I'm a noob in Docker.

Reply:
@poeti8 It's okay. :)